concept/type/RoleType.java

```diff
@@ -39,12 +39,6 @@ default boolean isRoleType() {
 
     interface Remote extends Type.Remote, RoleType {
 
-        @Override
-        RoleType getSupertype();
-
-        @Override
-        Stream<? extends RoleType> getSupertypes();
```

Comment:
Pretty sure we should keep these. These are still guaranteed to be `RoleType` and `Stream<? extends RoleType>`. The root RoleType is `relation:role` which is a `RoleType` and its supertype is `null` - it has no supertypes.